src/transformers/models/layoutlmv2/processing_layoutlmv2.py

```diff
@@ -99,54 +116,54 @@ def __call__(
 
         Please refer to the docstring of the above two methods for more information.
         """
+        output_kwargs = self._merge_kwargs(
+            LayoutLMv2ProcessorKwargs,
+            tokenizer_init_kwargs=self.tokenizer.init_kwargs,
+            **kwargs,
+        )
+
+        apply_ocr = output_kwargs["images_kwargs"].get("apply_ocr", self.image_processor.apply_ocr)
```

Comment:
sorry, didn't notice the first time it wasn't in kwargs initially. But seems okey to me allowing users to set all image kwargst from processor's call